flask_autodoc/templates/autodoc_default.html

```diff
@@ -21,6 +21,8 @@
             div.mapping {
                 margin: 20px 20px;
             }
+            
+            .location { font-style: italic; }
```

Comment:
Let's leave the default template as simple as possible (i.e. without location or custom properties). The changes to `examples/custom/templates/autodoc_custom.html` should be enough to explain how to use this new feature.